README.md

```diff
@@ -22,90 +23,47 @@
   - [Run it on iOS / Android](#3-run-it-on-ios--android)
 - [Credits](#credits)
 
-## Installation
-
-### 1. Integrate the base Facebook SDK
-
-Follow the instructions on [react-native-fbsdk](https://github.com/facebook/react-native-fbsdk) to integrate the Facebook SDK into your project.
-Note that for iOS, it's [recommended you use Cocoapods](https://developers.facebook.com/docs/ios/getting-started/advanced) rather than the manual installation.
-
-### 2. Integrate Facebook Audience Network
-
-#### For iOS:
-
-1. Run `pod install` in the `ios/` directory
-
-If you didn't use Cocoapods to integrate the Facebook SDK, you'll need to manually add the audience network framework file to your project.
+## Prerequisites
 
-#### For Android:
+You must have Facebook developer account in order to start integrating your app with this library. If you don't have one sign up [here](https://developers.facebook.com/).
 
-1. The library will automatically add audience-network-sdk to your dependencies. There is no need to add the audience network artifact to your build.gradle.
-
-2. In `MainApplication.java`, initialize the SDK inside `onCreate`:
-
-```java
-
-import suraj.tiwari.reactnativefbads.FBAdsPackage; // <-- add this
-
-public class MainApplication extends Application implements ReactApplication {
-...
-  @Override
-  public void onCreate() {
-    super.onCreate();
-  }
-  @Override
-  protected List<ReactPackage> getPackages() {
-    return Arrays.<ReactPackage>asList(
-          new FBAdsPackage()   // <-- add this
-    );
-  }
-...
-}
-```
+Follow the instructions on [react-native-fbsdk](https://github.com/facebook/react-native-fbsdk) to integrate the **Facebook SDK** into your project.
 
-### 3. Install The Javascript Package
+## Installation
 
-Add the package to your project using your favorite package manager
+Add the package to your project using your favorite package manager yarn:
 
 ```bash
 $ yarn add react-native-fbads
```

Comment:
Remove dollars, it's not beginners friendly and it makes it harder to copy whole thing easily
```suggestion
yarn add react-native-fbads
```